packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -255,6 +255,14 @@ export class ConnectionManager implements IConnectionManager {
 	public get clientId() {
 		return this.connection?.clientId;
 	}
+
+	private _connectionDetails?: IConnectionDetailsInternal;
+	public get connectionDetails() {
+		assert(this.connection !== undefined, "connected");
```

Comment:
Why do we have asserts here? It is just a public method to get the current connection details. Caller may not be bother about to call it only when we have a connection?